src/scss/application.scss

```
@@ -126,6 +126,180 @@
+.app-colour-key__swatch--inline {
+  width: 16px;
+  height: 16px;
+  margin: 0 4px;
+  vertical-align: middle;
+}
+
+.app-cell-changed {
+  background-color: #f8b26a;
+  box-shadow: inset 0 0 0 2px #d4692a;
+  cursor: help;
+  // Dotted underline cue so users know the cell has hover detail.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)